src/components/navbar/NavBar.tsx

```diff
@@ -10,56 +9,139 @@ import {
 } from "@mui/icons-material";
 import { signOut } from "next-auth/react";
 import { useSession } from "next-auth/react";
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import Toolbar from "@mui/material/Toolbar";
+
+export default function NavBar() {
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const drawer = (
+    <div className={$navbar.sidebar}>
+      <Link href="/">
+        <Image
+          src="/img/compass-logo-white.svg"
+          alt="logo"
+          className={$navbar.logo}
+          width={64}
+          height={64}
+          priority
+        />
+      </Link>
+      <List className={$navbar.list}>
+        <Link href="/students" className={$navbar.linkItem}>
+          <ListItem disablePadding className={$navbar.link}>
```

Comment:
Feels like the classnames should be swapped. link->link, linkItem->listItem. 

Also its technically invalid html for the `a` to be a direct child of `ul`. I think you did it this way so the the whole `li` would be clickable, but you could achieve the same by making the `a` `display: block` (or flex) and `width: 100%`.

Reply:
refer to [f7ecc99](https://github.com/sfbrigade/compass/pull/181/commits/f7ecc99da1360da8cb03ec8a37acd049cce07c13)